lib/themes/dosomething/paraneue_dosomething/js/campaign/HotModule.js

```diff
@@ -73,13 +73,15 @@ var createDateLabels = function(dates) {
   var dateLabels = [];
   var monthNames = ["Jan", "Feb", "Mar", "April", "May", "June", "July", "Aug", "Sept", "Oct","Nov", "Dec"];
 
-  dates.forEach(function(date, key) {
-    date = new Date(date);
-    var month = date.getMonth();
-    var day = date.getDate();
-
-    dateLabels[key] = monthNames[month] + " " + day;
-  });
+  if (dates) {
```

Comment:
It might be cleaner to make `dates` have a default value of `[]` so the `.forEach` just becomes a no-op.

```js
function createDateLabels(dates = []) {
 // ...
}
```